core/layers/svc-core-proxy/extension/server/serve.go

```
@@ -93,6 +95,27 @@ func handleRequest(getDetails func(ctx context.Context, tenantId string) (*model
 	}
 }
 
+// handleRequest is a generic handler for tenant-related requests
+func handleRequestWithId(getDetailsWithId func(ctx context.Context, tenantId, id string) (*models.Response, error)) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		tenantId := chi.URLParam(r, tenantIDParam)
+		id, err := url.QueryUnescape(chi.URLParam(r, "id"))
```

golangci-lint (1.62.2)

ineffectual assignment to err

(ineffassign)